src/libNode/Node.cpp

```diff
@@ -70,15 +70,25 @@ const unsigned int MIN_CHILD_CLUSTER_SIZE = 2;
 void addBalanceToGenesisAccount() {
   LOG_MARKER();
 
-  const uint128_t bal{std::numeric_limits<uint128_t>::max()};
+  const uint128_t balance_each = TOTAL_GENESIS_TOKEN / GENESIS_WALLETS.size();
+  const uint128_t balance_left = TOTAL_GENESIS_TOKEN % (GENESIS_WALLETS.size());
+
   const uint64_t nonce{0};
+  bool moduloCredited = false;
 
   for (auto& walletHexStr : GENESIS_WALLETS) {
     bytes addrBytes;
     if (!DataConversion::HexStrToUint8Vec(walletHexStr, addrBytes)) {
       continue;
     }
     Address addr{addrBytes};
+    uint128_t bal = 0;
+    if (!moduloCredited) {
```

Comment:
What is this logic doing?

Reply:
I think he is trying to put more balance that left by dividing to the first genesis account